src/parser/jobs/mnk/modules/Fists.js

```diff
@@ -68,12 +80,63 @@ export default class Fists extends Module {
 		this.addHook('complete', this._onComplete)
 	}
 
+	normalise(events) {
+		for (let i = 0; i < events.length; i++) {
+			const event = events[i]
+			const abilityId = event.ability.guid
```

Comment:
This could crash if `event.ability` isn't set on an event. Normaliser crashes will crash the entire application stack (intentionally) - let's avoid that.

Can't use  GH suggestions 'cus it's over multiple lines, but something along the lines of other normalisers would be

```js
if (!event.ability) { continue; }
const abilityId = event.ability.guid
```